src/CMakeLists.txt

```
@@ -18,6 +18,7 @@ add_library ("${PROJECT_NAME}_lib" STATIC modules/clustering/hierarchical_cluste
 target_link_libraries ("${PROJECT_NAME}_lib" PUBLIC seqan3::seqan3)
 target_link_libraries ("${PROJECT_NAME}_lib" PUBLIC fastcluster)
 target_link_libraries ("${PROJECT_NAME}_lib" PUBLIC bamit)
+target_link_libraries ("${PROJECT_NAME}_lib" PUBLIC bio)
```

```suggestion
target_link_libraries ("${PROJECT_NAME}_lib" PUBLIC bio::bio)
```
But if it works like this, it probably doesn't matter.

---

This results in an error:
```bash
CMake Error at src/CMakeLists.txt:4 (add_library):
  Target "iGenVar_lib" links to target "bio::bio" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
...
```

---

Yeah, this is expected to only work, if the find_package worked.